src/createFoundEnhancer.js

```diff
@@ -0,0 +1,35 @@
+import FarceActions from 'farce/lib/Actions';
+import queryMiddleware from 'farce/lib/queryMiddleware';
+import createHistoryEnhancer from 'farce/lib/createHistoryEnhancer';
+import { compose } from 'redux';
+
+import Matcher from './Matcher';
+import createMatchEnhancer from './createMatchEnhancer';
+
+export default function createFoundEnhancer({
```

Comment:
i'm really not sure about this API... it feels un-redux-y to me to bind a bunch of action dispatches to the store itself. it's not something that i've really seen

the `createHref` and `createLocation` things i do in `createHistoryEnhancer` in farce are because those are weird actions that are called not for side effects but for return values

can we perhaps unwind this API addition, and just do the `render` thing and the dep bumps?

Reply:
Maybe we can compromise? Something like:

```js
const store = createStore(
  combineReducers({
    found: foundReducer,
  }),
  compose(
    createBrowserHistoryEnhancer(),
    createDefaultMatchEnhancer(routeConfig, { matchStemRoutes: false }),
  ),
);
```

This would reduce the imports needed outside of the standard Redux imports to just two.

I don't think the import mess is too big an issue for dispatching actions, though, because that API is fairly foolproof.

We can also separate this from the changes here, perhaps? If you need anything urgently. Cutting new releases is pretty cheap.